internal/server/namespace.go

```diff
@@ -0,0 +1,25 @@
+package server
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/labstack/echo/v4"
+
+	"github.com/percona/everest/pkg/rbac"
+)
+
+// ListNamespaces returns the current version information.
+func (e *EverestServer) ListNamespaces(ctx echo.Context) error {
+	user, err := rbac.GetUser(ctx)
+	if err != nil {
+		return errors.Join(errFailedToGetUser, err)
+	}
+
+	result, err := e.handler.ListNamespaces(ctx.Request().Context(), user)
+	if err != nil {
+		e.l.Errorf("ListNamespaces failed: %w", err)
```

Comment:
Hmm, I think in that case, we don't have to chain the errors (i.e, do `return err` instead of `return errors.Join`), so the final error message is not so verbose.

But we can add a error log in each of the handlers so that we get a chain of log messages that allow us to pinpoint like a stack trace, wdyt?

Reply:
BTW, I planned to add adequate logging to the 3 handler implementations.. currently that is missing

Reply:
Yes, I agree, I thought the same

Reply:
If you don't to go through this in this PR that's fine, we'll need a follow-up PR for adding tests so we can use that